ddl/db_test.go

```diff
@@ -126,6 +126,8 @@ func setUpSuite(s *testDBSuite, c *C) {
 	c.Assert(err, IsNil)
 	_, err = s.s.Execute(context.Background(), "set @@global.tidb_max_delta_schema_count= 4096")
 	c.Assert(err, IsNil)
+	_, err = s.s.Execute(context.Background(), "set @@global.tidb_enable_alter_placement=1")
```

Comment:
I found the name `tidb_enable_alter_placement` is a little misleading now. Actually it means "wether enable placement or not", not just for alter.

Reply:
Because we'll finally remove the variable before GA, I think it's fine to keep its name unchanged.

Reply:
Discussed on Slack: Because the flag name is pre-existing, I think we should keep it. Otherwise, we'll end up with 2 flags. We will remove the flag once we GA anyway.